### ISSUE TITLE
Blank Encounter with Legendary Monster

### ISSUE THREAD

**Issue by VoltCruelerz** [etriebe/combat-estimate#10]
![image](https://user-images.githubusercontent.com/4068550/167516684-5143628c-3111-4912-96c6-60bab80577d8.png)

Console will be included below, but first the spells that aren't loading (even though I suspect they're unrelated problems):

### Frostbite (Brain Freeze)

![image](https://user-images.githubusercontent.com/4068550/167517077-bbfeb7d6-70fd-459a-a170-3cf37d66babb.png)
![image](https://user-images.githubusercontent.com/4068550/167517104-1e58a811-6281-4d34-b993-675df32d4dd4.png)

### Mind Sliver

![image](https://user-images.githubusercontent.com/4068550/167517195-96ee6d4d-d094-462d-8b37-4ca15b7ba752.png)
![image](https://user-images.githubusercontent.com/4068550/167517218-5fbee021-b31f-441b-94f0-85f29d3f047d.png)

### Gelatinous Titan Legendary Action
I can provide more info about this creature if needed, but I figured this was the place to start since legendary action support was just added.
![image](https://user-images.githubusercontent.com/4068550/167517267-af5fcdb0-08ca-4766-9f91-4691f0d310eb.png)
![image](https://user-images.githubusercontent.com/4068550/167517342-85a0b8b0-d320-4f3d-82a4-fb7be8776ab2.png)

### Console
```js
Foundry VTT | Rendering CombatEstimateDialog
dialog.js:71 Combatant 1469 is friendly
ActorUtils.js:550 Unable to parse spell attack Frostbite
getSpellDataPerRound @ ActorUtils.js:550
PCActor5e @ PCActor5e.js:14
getPCActorObject @ ActorUtils.js:62
populateCombatants @ dialog.js:72
activateListeners @ dialog.js:37
_render @ foundry.js:3090
async function (async)
_render @ foundry.js:3062
_render @ foundry.js:3764
render @ foundry.js:3021
(anonymous) @ module.js:32
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
ActorUtils.js:551 SyntaxError: Invalid or unexpected token
    at Function.getAverageDamageFromDescription (ActorUtils.js:1091)
    at Function.getInfoForSpellObject (ActorUtils.js:760)
    at Function.getSpellDataPerRound (ActorUtils.js:531)
    at new PCActor5e (PCActor5e.js:14)
    at Function.getPCActorObject (ActorUtils.js:62)
    at CombatEstimateDialog.populateCombatants (dialog.js:72)
    at CombatEstimateDialog.activateListeners (dialog.js:37)
    at CombatEstimateDialog._render (foundry.js:3090)
    at async CombatEstimateDialog._render (foundry.js:3764)
getSpellDataPerRound @ ActorUtils.js:551
PCActor5e @ PCActor5e.js:14
getPCActorObject @ ActorUtils.js:62
populateCombatants @ dialog.js:72
activateListeners @ dialog.js:37
_render @ foundry.js:3090
async function (async)
_render @ foundry.js:3062
_render @ foundry.js:3764
render @ foundry.js:3021
(anonymous) @ module.js:32
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
ActorUtils.js:550 Unable to parse spell attack Frostbite (Brain Freeze)
getSpellDataPerRound @ ActorUtils.js:550
PCActor5e @ PCActor5e.js:14
getPCActorObject @ ActorUtils.js:62
populateCombatants @ dialog.js:72
activateListeners @ dialog.js:37
_render @ foundry.js:3090
async function (async)
_render @ foundry.js:3062
_render @ foundry.js:3764
render @ foundry.js:3021
(anonymous) @ module.js:32
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
ActorUtils.js:551 SyntaxError: Invalid or unexpected token
    at Function.getAverageDamageFromDescription (ActorUtils.js:1091)
    at Function.getInfoForSpellObject (ActorUtils.js:760)
    at Function.getSpellDataPerRound (ActorUtils.js:531)
    at new PCActor5e (PCActor5e.js:14)
    at Function.getPCActorObject (ActorUtils.js:62)
    at CombatEstimateDialog.populateCombatants (dialog.js:72)
    at CombatEstimateDialog.activateListeners (dialog.js:37)
    at CombatEstimateDialog._render (foundry.js:3090)
    at async CombatEstimateDialog._render (foundry.js:3764)
getSpellDataPerRound @ ActorUtils.js:551
PCActor5e @ PCActor5e.js:14
getPCActorObject @ ActorUtils.js:62
populateCombatants @ dialog.js:72
activateListeners @ dialog.js:37
_render @ foundry.js:3090
async function (async)
_render @ foundry.js:3062
_render @ foundry.js:3764
render @ foundry.js:3021
(anonymous) @ module.js:32
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
ActorUtils.js:550 Unable to parse spell attack Mind Sliver
getSpellDataPerRound @ ActorUtils.js:550
PCActor5e @ PCActor5e.js:14
getPCActorObject @ ActorUtils.js:62
populateCombatants @ dialog.js:72
activateListeners @ dialog.js:37
_render @ foundry.js:3090
async function (async)
_render @ foundry.js:3062
_render @ foundry.js:3764
render @ foundry.js:3021
(anonymous) @ module.js:32
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
ActorUtils.js:551 SyntaxError: Invalid or unexpected token
    at Function.getAverageDamageFromDescription (ActorUtils.js:1091)
    at Function.getInfoForSpellObject (ActorUtils.js:760)
    at Function.getSpellDataPerRound (ActorUtils.js:531)
    at new PCActor5e (PCActor5e.js:14)
    at Function.getPCActorObject (ActorUtils.js:62)
    at CombatEstimateDialog.populateCombatants (dialog.js:72)
    at CombatEstimateDialog.activateListeners (dialog.js:37)
    at CombatEstimateDialog._render (foundry.js:3090)
    at async CombatEstimateDialog._render (foundry.js:3764)
getSpellDataPerRound @ ActorUtils.js:551
PCActor5e @ PCActor5e.js:14
getPCActorObject @ ActorUtils.js:62
populateCombatants @ dialog.js:72
activateListeners @ dialog.js:37
_render @ foundry.js:3090
async function (async)
_render @ foundry.js:3062
_render @ foundry.js:3764
render @ foundry.js:3021
(anonymous) @ module.js:32
dispatch @ jquery.min.js:2
v.handle @ jquery.min.js:2
dialog.js:79 Combatant Gelatinous Titan is hostile
appbar.js:54 CombatEstimateDialog {options: {…}, appId: 126, _element: S.fn.init(1), position: {…}, _dragDrop: Array(0), …}
FoundryUtils.js:75 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'data')
[Detected 1 package: combat-estimate]
    at Function.getDataObjectFromObject (FoundryUtils.js:75)
    at Function.getLegendaryActions (ActorUtils.js:283)
    at new NPCActor5e (NPCActor5e.js:18)
    at Function.getActorObject (ActorUtils.js:44)
    at CombatEstimateDialog.populateCombatants (dialog.js:80)
    at CombatEstimateDialog.activateListeners (dialog.js:37)
    at CombatEstimateDialog._render (foundry.js:3090)
    at async CombatEstimateDialog._render (foundry.js:3764)
```

**Comment by etriebe** [comment]
Would you mind exporting those spells and creatures into a json file for me to import? 

Or grant me GM access to your world?

**Comment by etriebe** [comment]
Feel free to DM me on discord too (mentioned in module page).

**Comment by etriebe** [comment]
@VoltCruelerz is this the gelatinous titan and did you import with Mr Primate's D&D Beyond importer?

https://www.dndbeyond.com/monsters/392380-gelatinous-titan

**Comment by etriebe** [comment]
https://github.com/etriebe/combat-estimate/releases/tag/1.0.12

I believe I fixed for both issues that were happening in 1.0.12. Pushed release.
